### DISCUSSION TITLE
Doesn't Cover the entire 4k secondary monitor

### DISCUSSION THREAD

**Discussion by mmc-terraform** [pubpub-zz/ppInk#193] [General]
Ppink only covers a part of the secondary monitor. My primary monitor is QHD 2560x1440 and my secondary monitor is 4k. Not sure if that is causing this issue.

https://user-images.githubusercontent.com/79689065/177726585-ca5bdbe7-a949-4c13-b284-49bb2aedeaf8.mp4

**Comment by pubpub-zz** [comment]
Hi @mmc-terraform ,
This most probably linked with issues with multiple screens resolution have a look at tips at the bottom of https://github.com/pubpub-zz/ppInk

**Comment by pubpub-zz** [comment]
@mmc-terraform is it ok now ?
